### PR TITLE
cli: inline evaluate

### PR DESCRIPTION
**Pull request by slavaGanzin**
Now you can use cli for inline interpretation:
```
bin/cli.js script1 script2
> 123 + 3
126
> + 15
15
> 
vganzin ~/work/mathjs bin/cli.js 1+2
3
```

p.s. I can't find any cli tests

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -420,9 +422,14 @@ else if (scripts.length === 0) {
   runStream(process.stdin, process.stdout, mode, parenthesis);
 }
 else {
-  //work through the queue of scripts
-  scripts.forEach(function (arg) {
-    // run a script file
-    runStream(fs.createReadStream(arg), process.stdout, mode, parenthesis);
-  });
+  fs.stat(scripts[0], (e, f) => {
```
Comment (josdejong) [inline bin/cli.js]: Can you please replace the arrow function with a regular function so the cli also works with old version of node.js?

File: bin/cli.js [hunk 2]
```diff
@@ -420,9 +422,14 @@ else if (scripts.length === 0) {
...
+    if (e)
```
Comment (josdejong) [inline bin/cli.js]: I personally prefer explicit if/else and always use code blocks like `if (...) {...} else {....}` for more robust code, but it's not a big deal.

Comment (slavaGanzin) [inline bin/cli.js]: Review again please

**Comment by josdejong** [comment]
Thanks a lot Slava!

**Pull request merged**